src/components/TimerControls.tsx

```diff
@@ -9,9 +12,15 @@ const TimerControls = ({
 }): JSX.Element => {
 	return (
 		<>
-			<button type="button" onClick={pauseTimer}>
-				{isTimerPaused ? "Resume" : "Pause"}
-			</button>
+			{isTimerPaused ? (
+				<PlayButton onClick={pauseTimer}>
```

Comment:
It might be a little more readable to name this `<ResumeButton>` instead of PlayButton since that is the language we use elsewhere in the code